moped-database/views/project_list_view.sql

```diff
@@ -236,7 +274,9 @@ SELECT
         GROUP BY ptags.project_id
     ) AS project_tags,
     concat(added_by_user.first_name, ' ', added_by_user.last_name) AS added_by,
-    mpcs.components
+    mpcs.components,
+    districts.self_alone_council_districts,
+    districts.self_and_children_council_districts
```

Comment:
I went real explicit on these column names because it's an uncommon distinction to be making, and I'm _very_ open to suggestions for names.

Reply:
how about: `project_council_districts` and `project_and_child_project_council_districts` ?